src/main/resources/docs/description/Generic_Files_ExecutableFile.md

```diff
@@ -0,0 +1 @@
+Files should not be executable.
```

Comment:
Is this correct?

Reply:
Yes, https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/Generic/Sniffs/Files/ExecutableFileSniff.php